src/main/ApplicationImpl.cpp

```diff
@@ -550,15 +550,7 @@ ApplicationImpl::getJsonInfo(bool verbose)
 void
 ApplicationImpl::reportInfo(bool verbose)
 {
-    auto loadConfig = [this]() {
-        LedgerTxn ltx(getLedgerTxnRoot());
-        if (protocolVersionStartsFrom(ltx.loadHeader().current().ledgerVersion,
-                                      SOROBAN_PROTOCOL_VERSION))
-        {
-            getLedgerManager().updateNetworkConfig(ltx);
-        }
-    };
-    mLedgerManager->loadLastKnownLedger(loadConfig);
+    mLedgerManager->loadLastKnownLedger(false, true);
```

Comment:
Nit: add `/*startSevices=*/` and `/*ledgerStateReady=*/` so we don't have back to back to back raw bools. Applies in a few other places as well.